<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchRecord</class>
 <widget class="QWidget" name="SearchRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>341</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>392</width>
    <height>341</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>392</width>
    <height>341</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Поиск</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>97</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IDLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>201</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите данные, по которым будет производиться поиск</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="weekdayLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>201</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>День недели</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>201</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Дата</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="monthLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>201</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Месяц</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IDradioButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="weekdayRadioButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="dateRadioButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="monthRadioButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>97</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Сброс</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
